client/pages/AssessmentPrint/AssessmentPrint.tsx

```diff
@@ -0,0 +1,41 @@
+import './style.scss'
+import React, { memo, useEffect } from 'react'
+
+import { AssessmentName } from '@meta/assessment'
+
+import { useAppDispatch } from '@client/store'
+import { AssessmentActions, useAssessment, useCycle } from '@client/store/assessment'
+import { useCountryIso } from '@client/hooks'
+import Loading from '@client/components/Loading'
+
+import FraPrint from './FraPrint'
+
+const Components: Record<string, React.FC> = {
+  [AssessmentName.fra]: FraPrint,
+}
+
+const AssessmentPrint: React.FC = () => {
+  const dispatch = useAppDispatch()
+  const assessment = useAssessment()
+  const cycle = useCycle()
+  const countryIso = useCountryIso()
+  // useInitCountry()
+  // const assessmentType = useAssessmentType()
+  // const countryStatusLoaded = useIsCountryStatusLoaded()
+  const Component = Components[assessment?.props.name]
+
+  useEffect(() => {
+    dispatch(AssessmentActions.initApp())
```

Comment:
Why is initApp needed here ?

Reply:
PrintView has no AppInit call (opens in separate window/tab)

Reply:
Removed - only sections need to be fetched